kernels/portable/cpu/pattern/pattern.h

```diff
@@ -84,11 +84,18 @@ Tensor& unary_ufunc_realhb_to_bool(
  * the math operation which is applied to the input tensor element-wise.
  */
 Tensor& unary_ufunc_realhbbf16_to_floathbf16(
-    double (*fn)(double),
+    float (*fn_float)(float),
+    double (*fn_double)(double),
     KernelRuntimeContext& ctx,
     const Tensor& in,
     Tensor& out);
 
+#define DEFINE_UNARY_UFUNC_REALHBBF16_TO_FLOATHBF16(op_name, fn)              \
+  Tensor& op_name(KernelRuntimeContext& ctx, const Tensor& in, Tensor& out) { \
+    return internal::unary_ufunc_realhbbf16_to_floathbf16(                    \
+        fn, fn, ctx, in, out);                                                \
+  }
+
```

Comment:
why not define this macro to use the full namespace (prefixed with :: so it works from anywhere) so that backends/cadence can use it too?